x-pack/plugins/data_visualizer/public/application/index_data_visualizer/components/search_panel/search_panel.tsx

```diff
@@ -120,7 +120,7 @@ export const SearchPanel: FC<Props> = ({
   return (
     <EuiFlexGroup
       gutterSize="s"
-      alignItems="flexStart"
+      alignItems="center"
```

Comment:
@stratoula Since we want the the additional filters to flow down without changing the positions of the other controls, I think `center` is not what we want here. I think applying the following patch should resolve this styling issue.

```
Index: x-pack/plugins/data_visualizer/public/application/index_data_visualizer/components/search_panel/search_panel.tsx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- x-pack/plugins/data_visualizer/public/application/index_data_visualizer/components/search_panel/search_panel.tsx	(revision 7074fc14a6951bb0404407f475037db9b517b241)
+++ x-pack/plugins/data_visualizer/public/application/index_data_visualizer/components/search_panel/search_panel.tsx	(date 1651079563850)
@@ -120,7 +120,6 @@
   return (
     <EuiFlexGroup
       gutterSize="s"
-      alignItems="center"
       data-test-subj="dataVisualizerSearchPanel"
       className={'dvSearchPanel__container'}
       responsive={false}
Index: x-pack/plugins/data_visualizer/public/application/index_data_visualizer/components/search_panel/search_panel.scss
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- x-pack/plugins/data_visualizer/public/application/index_data_visualizer/components/search_panel/search_panel.scss	(revision 7074fc14a6951bb0404407f475037db9b517b241)
+++ x-pack/plugins/data_visualizer/public/application/index_data_visualizer/components/search_panel/search_panel.scss	(date 1651079622027)
@@ -3,6 +3,10 @@
   padding: $euiSizeS;
 }
 
+.dvSearchPanel__container {
+  align-items: baseline;
+}
+
 @include euiBreakpoint('xs', 's', 'm', 'l') {
   .dvSearchPanel__container {
     flex-direction: column;
```

Reply:
@qn895 I committed your fix. Thanx for that! How do you feel about Caroline's suggestion?

Reply:
Thanks for adding the fix @stratoula. @cchaos I agree that it's not ideal... But for the purpose of this PR I think it's best to leave the behavior as close to what we currently have on main. This behavior is definitely something we should revisit in 8.3 as a follow up after this PR is merged if that sounds good to ya'll.